<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkMonitorDialog</class>
 <widget class="QDialog" name="NetworkMonitorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Monitor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Request Headers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="requestDetailsView">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Response Headers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="responseDetailsView">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Network Requests</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="clearButton">
     <property name="text">
      <string>&amp;Clear</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableView" name="requestList">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>splitter</zorder>
  <zorder>label</zorder>
  <zorder>clearButton</zorder>
  <zorder>requestList</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
